diff -Naur a/bind/Makefile.in b/bind/Makefile.in
--- a/bind/Makefile.in	2018-07-02 11:12:19.897979319 -0400
+++ b/bind/Makefile.in	2018-07-02 11:39:25.843115125 -0400
@@ -77,19 +77,13 @@
 		echo Bind libraries already installed ;                   \
 	else                                                              \
 		echo Building BIND libraries - this takes some time. ;    \
-		for libdir in ${bindlibs} ; do                            \
-		 (cd ${bindsrcdir}/lib/$$libdir ;                         \
-		  echo Building $$libdir library in `pwd` ;               \
-		  $(MAKE) all >> ${binddir}/build.log) ;                      \
-		done ;                                                    \
+		 (cd ${bindsrcdir}/lib/dns ;                              \
+		  echo Building gen using ${BUILD_CC} in `pwd` ;          \
+		  $(MAKE) CC=${BUILD_CC} CFLAGS="-O2" LIBS="" gen) ;      \
 		                                                          \
 		echo Installing BIND libraries to ${binddir}. ;           \
-		for libdir in ${bindlibs} ; do                            \
-		 (cd ${bindsrcdir}/lib/$$libdir ;                         \
-		  MAKEDEFS="${installdirs}"; export MAKEDEFS;             \
-		  $(MAKE) ${installdirs} LIBTOOL_MODE_INSTALL= install >> \
-		   ${binddir}/install.log) ;                              \
-		done ;                                                    \
+		 (cd ${bindsrcdir}/lib/dns ;                              \
+		   $(MAKE) DESTDIR="" install > ${binddir}/install.log) ; \
 	fi
 
 clean:
@@ -97,7 +91,7 @@
 	rm -rf ${bindsrcdir} ${cleandirs} ${cleanfiles}
 
 @INSTALL_BIND_FALSE@install:
-@INSTALL_BIND_TRUE@install: install-bind
+@INSTALL_BIND_TRUE@install:
 
 install-bind: all
 	@for libdir in ${bindlibs} ; do   \
