diff -Naur a/bind/Makefile.in b/bind/Makefile.in
--- a/bind/Makefile.in	2018-07-02 11:39:25.843115125 -0400
+++ b/bind/Makefile.in	2018-07-02 20:07:55.487829926 -0400
@@ -21,8 +21,10 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 
-bindconfig = --without-openssl --without-libxml2 --without-libjson \
-	--without-gssapi --disable-threads --without-lmdb \
+bindconfig = --disable-atomic --disable-kqueue --disable-epoll \
+	--disable-devpoll --without-openssl --without-libxml2 \
+	--without-libjson --without-gssapi --disable-threads \
+	--without-lmdb --without-randomdev \
 	--includedir=@includedir@ --libdir=@libdir@ \
 	@BINDLT@ @BINDIOMUX@ @BINDCONFIG@ --enable-full-report
 
@@ -55,7 +57,7 @@
 	else                                                            \
 		echo Configuring BIND libraries for DHCP. ;             \
 		rm -rf ${cleandirs} ${cleanfiles} ;                     \
-		(cd ${bindsrcdir} &&                                    \
+               (cd ${bindsrcdir} && export CC=${CROSS_CC} &&           \
                  ./configure ${bindconfig} > ${binddir}/configure.log); \
 	fi
 
