diff -Naur a/bind/Makefile.in b/bind/Makefile.in
--- a/bind/Makefile.in	2018-07-03 10:12:46.982988456 -0400
+++ b/bind/Makefile.in	2018-07-03 10:54:03.091672845 -0400
@@ -21,8 +21,10 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 
-bindconfig = --without-openssl --without-libxml2 --without-libjson \
-	--without-gssapi --disable-threads --without-lmdb \
+bindconfig = --disable-atomic --disable-kqueue --disable-epoll \
+	--disable-devpoll --without-openssl --without-libxml2 \
+	--without-libjson --without-gssapi --disable-threads \
+	--without-lmdb --without-randomdev \
 	--includedir=@includedir@ --libdir=@libdir@ \
 	@BINDLT@ @BINDIOMUX@ @BINDCONFIG@ --enable-full-report
 
@@ -31,7 +33,7 @@
 cleanfiles = ./configure.log ./build.log ./install.log
 
 bindlibs = isc dns isccfg irs
-installdirs = includedir=${binddir}/include libdir=${binddir}/lib
+installdirs = includedir=/../bind/include libdir=/../bind/lib
 
 @BIND_ATF_FALSE@all: bind1 bind2
 @BIND_ATF_TRUE@all: bind1 atf bind2
@@ -105,6 +107,8 @@
 	  $(MAKE) install) ;              \
 	 done
 
+install-exec:
+
 @INSTALL_BIND_FALSE@uninstall:
 @INSTALL_BIND_TRUE@uninstall: uninstall-bind
 
